dockyard/common/container/container.py

```diff
@@ -0,0 +1,162 @@
+from dockyard.common import utils, link
+
+import json
+from pecan.core import redirect
+
+class Container(object):
+    def __init__(self):
+        pass
+
+    def list(self, name_id=None, query_params=None):
+        if name_id:
+            url = ('/containers/%s/json' % (name_id))
+        else:
+            url = '/containers/json'
+
+        if query_params:
+            url = ('%s?%s' % (url, query_params))
+
+        return utils.dispatch_get_request(url, 'http').data
+
+    def changes(self, name_id=None):
+        url = (('/containers/%s/changes') % (name_id))
+        return utils.dispatch_get_request(url, 'http').data
+
+    def resize(self, _id, **kwargs):
+        url = (('/containers/%s/changes') % (_id))
+
+        if kwargs:
+            query = link.make_query_url(kwargs)
+            url = ('%s?%s' % (url, query ))
+
+        return utils.dispatch_get_request(url, 'http').data
+
+    def export(self, name_id=None):
+        url = (('/containers/%s/export') % (name_id))
+        redirect(utils.get_link(url, 'http'))
+        # return str(utils.dispatch_get_request(url, 'http').data)
+
+    def top(self, name_id=None, query_params=None):
+        url = (('/containers/%s/top') % (name_id))
+
+        if query_params:
+            url = ('%s?%s' % (url, query_params))
+
+        return utils.dispatch_get_request(url, 'http').data
+
+    def stats(self, _id):
+        url = ('/containers/%s/stats' % (_id))
+        redirect(utils.get_link(url, 'http'))
+        # return utils.dispatch_get_request(url, 'http')
+
+    def archive(self, _id, query_params=None):
```

Comment:
A huge amount of data is being get in the response and it is being passed therefor we should make sure that it scales well in the production environment.